Clang build issue: Undefined symbols for architecture x86_64 _IOPSGetTimeRemainingEstimate

Sorry for spamming issues, but after solving the first two issues and adding openssl include dir to INTERFACE_INCLUDE_DIRECTORIES I get the following:

```
[ 84%] Linking CXX executable monero-lws-admin
Undefined symbols for architecture x86_64:
  "_IOPSGetTimeRemainingEstimate", referenced from:
      cryptonote::miner::background_worker_thread() in libcryptonote_basic.a(miner.cpp.o)
      cryptonote::miner::on_battery_power() in libcryptonote_basic.a(miner.cpp.o)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [src/monero-lws-admin] Error 1
make[1]: *** [src/CMakeFiles/monero-lws-admin.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -205,6 +205,11 @@ foreach (LIB ${MONERO_LIBRARIES})
   unset(LIB_PATH CACHE)
 endforeach()
 
+if(APPLE)
+  find_library(IOKIT_LIBRARY IOKit)
+  list(APPEND IMPORTED_MONERO_LIBRARIES ${IOKIT_LIBRARY})
+endif()
+
 add_library(monero::libraries INTERFACE IMPORTED)
 set_property(TARGET monero::libraries PROPERTY
   INTERFACE_INCLUDE_DIRECTORIES

